src/main/java/bio/terra/service/load/LoadDao.java

```diff
@@ -18,12 +17,12 @@
 import java.util.UUID;
 import java.util.stream.Stream;
 import org.apache.commons.codec.binary.StringUtils;
+import org.jetbrains.annotations.NotNull;
```

Comment:
It looks like the TDR code currently uses `javax.validation.constraints.Nonnull` for this kind of annotation.

Although I'm not sure how much benefit we get from adding it, as IntelliJ can infer it. Was it suggested by sonar?

Reply:
IntelliJ flagged this as a nullability problem, and did not consider it solved with the javax annotation:

<img width="539" alt="Screenshot 2023-08-18 at 5 26 38 PM" src="https://github.com/DataBiosphere/jade-data-repo/assets/79769153/e5abc116-a9b5-4329-abf6-08c3cbe6853f">


<img width="901" alt="Screenshot 2023-08-18 at 5 24 58 PM" src="https://github.com/DataBiosphere/jade-data-repo/assets/79769153/aa3544ee-7bf1-43e0-989d-376e621350ca">

Reply:
I have this enabled too. Intellij does let you customize the annotations it recognizes for `Nullable` and `NotNull`, maybe I added the `javax` one to the list by hand?

![image](https://github.com/DataBiosphere/jade-data-repo/assets/1799360/e19bc4dd-ac6e-4664-a8a1-36dcca7a5827)